test/image/mocks/gl2d_parcoords_style_labels.json

```diff
@@ -19,7 +19,37 @@
             },
             "dimensions": [
                 {
-                    "label": "$\\varphi ={\\frac {1+{\\sqrt {5}}}{2}}$",
+                    "label": "<i>Rien ne se perd,<br>rien ne se crée,<br>tout se <b>transforme</b>.</i><sub>",
```

Comment:
Hmm. So parcoords trace with MathJax text don't render ok on CI?

Reply:
Actually they do render on the CI. But I noticed the mock often fails locally when testing multiple parcoords. By the way there could be an issue with a callback required for MathJax.

Reply:
Hmm. That's weird. Looks like you're using `convertToTspans` correctly - which should be filling in [`gd._promises`](https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/lib/svg_text_utils.js#L77).

I suspect the problem is in here:

https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/traces/parcoords/base_plot.js#L35-L67